chatrix.php

```diff
@@ -8,15 +8,25 @@
  * Version: 0.2.1
  */
 
-const AUTOMATTIC_CHATRIX_VERSION = '0.2.1';
-
 use function Automattic\Chatrix\Admin\main as adminMain;
 use function Automattic\Chatrix\main;
 
 if ( ! function_exists( 'Automattic\Chatrix\main' ) ) {
 	require __DIR__ . '/vendor/autoload.php';
 }
 
+function automattic_chatrix_version() {
+	if ( defined( 'WP_DEBUG' ) && WP_DEBUG === true ) {
+		// So that assets aren't cached in development environments.
+		return null;
```

Comment:
For better cache busting in dev environments, you can consider returning value of `time()` here ensuring cache is always busted with each page load.

Reply:
Makes sense!